docker-compose.yaml

```diff
@@ -1,13 +1,15 @@
 # Docker compose based on official Airflow example, available here:
-# https://airflow.apache.org/docs/apache-airflow/2.6.1/docker-compose.yaml
-# and documented here:
+# https://github.com/apache/airflow/blob/2.10.5/docs/apache-airflow/howto/docker-compose/docker-compose.yaml
+
+# And which is documented here:
 # https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#running-airflow-in-docker
+
 x-airflow-common: &airflow-common
-  # In order to add custom dependencies or upgrade provider packages you can use your extended image.
-  # Comment the image line, place your Dockerfile in the directory where you placed the docker-compose.yaml
-  # and uncomment the "build" line below, Then run `docker compose build` to build the images.
-  # build: .
-  image: atddocker/atd-airflow:production
+  # the `image:` key here is used in production to pull our published, production airflow image.
+  # the `build:` key is used in development to build the image locally.
+  build: .
+  # image: atddocker/atd-airflow:production
```

Comment:
@frankhereford are you going to switch back to image here before merge?

Reply:
Goodness, thank you Mike. yes!